dask/array/linalg.py

```diff
@@ -250,13 +251,13 @@ def tsqr(data, compute_svd=False, _max_vchunk_size=None):
         dependencies[name_q_st2] = q_inner.__dask_layers__()
 
         # R: R_inner
-        name_r_st2 = "r-inner-" + token
+        name_r_st2 = "r-inner" + token
```

Comment:
Why remove the dash before the token?

Reply:
The `token` variable has a dash in it already, so we were getting double-dashes.